app/main.py

```diff
@@ -1,6 +1,27 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+load_dotenv()
+
+API_KEY = os.getenv("API_KEY")
+URL = "http://api.weatherapi.com/v1/current.json"
```

Comment:
The `URL` constant should be defined in uppercase style to follow the convention for constants, which improves code readability and maintainability.